### ISSUE TITLE
Eliminate spurious Pytest warnings about unregistered marks

### ISSUE THREAD

**Issue by jaywritescode** [exercism/python#2745]
When running the test suites for a Python exercise in a local environment, Pytest generates a set of warnings because each of the tests is annotated (decorated) with `@pytest.mark.task`. Because `task` isn't [registered](https://docs.pytest.org/en/latest/how-to/mark.html#registering-marks), the Pytest output includes lines such as
```
dicts_test.py:50
  /Users/xyz/Exercism/python/inventory-management/dicts_test.py:50: PytestUnknownMarkWarning: Unknown pytest.mark.task - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.task(taskno=4)
```
This output is noisy and of little value to the developer.

Would it be reasonable to add a `pytest.ini` or `pyproject.toml` file as described in the link above to each of the exercises?

**Comment by jaywritescode** [comment]
Taking a second look, there is the appropriate `pytest.ini` file in this repo; however, it's not being copied into the individual exercise packages.

**Comment by BethanyG** [comment]
Hi @jaywritescode,

Thanks for logging this issue.  The warnings should vanish if you place the `ini` file in the root of the exercise folder (e.g. under `python/`.  We're investigating why it isn't currently downloading with the exercises.

Additionally, you can run pytest with `pytest -o markers=task` or use the `PYTEST_ADOPTS` env var to get the warnings to disappear.

**Comment by BethanyG** [comment]
Flagging this as a bug, since we haven't (yet) figured out how to get this to download with the CLI.

**Comment by BethanyG** [comment]
After discussing this with @ErikSchierboom, the ultimate resolution will be to place our `pytest.ini` file in `python/exercises/shared/`, so that it can be dowloaded via the CLI.  

However, that download behavior is not yet supported, so in the interim, we will have setup notes/instructions for students to download the file from the repo to copy into their root `python/` folder, or have them create  an `ini` file in that location.

**Comment by BethanyG** [comment]
via PR #2806, made a copy of the `pytest.ini` file in `exercises/shared`.  Testing docs have been updated.  Once the file is downloadable via CLI, we can close this issue.

**Comment by BethanyG** [comment]
Linking closed issue #2539, as the documentation is still not ideal around this issue.  Will be opening separate issues for updating documentation to make it clearer what needs to happen for students to set this up properly.

**Comment by BethanyG** [comment]
@ErikSchierboom  -- tagging you because we discussed a while ago having the `pytest.ini` download via the CLI, and I'm  not sure where we are on that front.  (_no worries if there is a delay -- I just don't want to forget about it._)

I have updated our testing documentation, and that has helped with this issue a lot -- but would also love to have the config file available via CLI in `exercises/shared` if possible... just let me know.  Thanks!  😄

**Comment by ErikSchierboom** [comment]
Yeah, sorry, that has not yet been implemented. I'll let you know when I get to that (such a huge list of TODOs :D)

**Comment by BethanyG** [comment]
Tagging this as officially "on hold" for the auto-download option.  Since we've changed the docs to clarify, this feels somewhat lower-priority at this stage.
